lib/capybara/webkit/node.rb

```diff
@@ -68,16 +68,19 @@ def unselect_option
       end
     end
 
-    def click
-      invoke("leftClick")
+    def click(*keys)
+      offset = keys.last.is_a?(Hash) ? keys.pop : {}
```

Comment:
Are these arguments coming from Capybara? Would it be possible for Capybara to handle the argument parsing and then provide well-formed arguments to drivers?

Reply:
The issue is supporting Capybara < 3 which I assume will still be wanted.  It doesn't pass any options to `click` so the driver needs to accept with or without

Reply:
Could it use default arguments? Something like: `click(keys = [], offset = {})`

I guess it's the varargs I'm worried about.

Reply:
Yes it definitely can -- not sure why I was overthinking it -- and that's what review is for -- thanks.